plugins/cairo-lang-macro/src/lib.rs

```diff
@@ -164,3 +164,11 @@ pub unsafe extern "C" fn post_process_callback(
     }
     context
 }
+
+/// A no-op Cairo attribute macro implementation.
+///
+/// This macro implementation does not produce any changes.
+/// Can be exposed as a placeholder macro for the internal purposes.
+pub fn no_op_attr(_attr: TokenStream, input: TokenStream) -> ProcMacroResult {
```

Comment:
If this is for internal use - shouldn't this be `#[doc(hidden)]`?

Reply:
Sure, sorry!